modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -194,35 +133,45 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: kubevirt-vm-latency-checkup
-  namespace: kiagnose
 spec:
   backoffLimit: 0
   template:
     spec:
-      serviceAccount: kiagnose
+      serviceAccountName: vm-latency-checkup-sa
       restartPolicy: Never
       containers:
-        - name: framework
+        - name: vm-latency-checkup
           image: registry.redhat.io/container-native-virtualization/checkup-framework:v4.12.0
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: ["ALL"]
+            runAsNonRoot: true
+            seccompProfile:
+              type: "RuntimeDefault"
           env:
             - name: CONFIGMAP_NAMESPACE
               value: <target_namespace>
             - name: CONFIGMAP_NAME
               value: kubevirt-vm-latency-checkup-config
+            - name: POD_UID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.uid
```

Comment:
The `POD_UID` environment variable is not supported in 4.12.
It will be relevant in the next version.
```suggestion
```